lib/ExpensiMark.js

```diff
@@ -731,15 +740,20 @@ export default class ExpensiMark {
 
     /**
      * @param {String} comment
-     * @returns {Array}
+     * @returns {Array} or undefined if exception occurs when executing regex matching
      */
     extractLinksInMarkdownComment(comment) {
-        const escapedComment = _.escape(comment);
-        const matches = [...escapedComment.matchAll(MARKDOWN_LINK_REGEX)];
-
-        // Element 1 from match is the regex group if it exists which contains the link URLs
-        const links = _.map(matches, match => Str.sanitizeURL(match[2]));
-        return links;
+        try {
+            const escapedComment = _.escape(comment);
+            const matches = [...escapedComment.matchAll(MARKDOWN_LINK_REGEX)];
+
+            // Element 1 from match is the regex group if it exists which contains the link URLs
+            const links = _.map(matches, match => Str.sanitizeURL(match[2]));
+            return links;
+        } catch (e) {
+            Log.warn('Exception occurs in method ExpensiMark.extractLinksInMarkdownComment', {error: e});
```

Comment:
Let's update this to be a bit more specific...

```
Log.warn('Error parsing url in ExpensiMark.extractLinksInMarkdownComment', {error: e});
```

Reply:
Updated